modules/chemical_reactions/src/kernels/CoupledBEEquilibriumSub.C

```diff
@@ -41,11 +41,11 @@ CoupledBEEquilibriumSub::CoupledBEEquilibriumSub(const InputParameters & paramet
     _sto_v(getParam<std::vector<Real>>("sto_v")),
     _gamma_u(coupledValue("gamma_u")),
     _gamma_u_old(coupledValueOld("gamma_u")),
-    _gamma_v(isCoupled("gamma_v")
+    _gamma_v(coupledComponents("gamma_v") > 1
```

Comment:
`isCoupled` will return false if none of the supplied input file values are variables (e.g. `gamma_v = '1 2 3 4'`)!

Reply:
Care to improve the doxygen comment of this function in the header file?

Reply:
Done